src/components/update/editor.tsx

```diff
@@ -41,7 +41,7 @@ const UpdatesEditor = ({
 }: UpdatesEditorProps) => {
   const router = useRouter();
   const { toast } = useToast();
-  const baseUrl = env("NEXT_PUBLIC_BASE_URL");
+  const baseUrl = NEXT_PUBLIC_BASE_URL();
```

Comment:
Can change this to just `NEXT_PUBLIC_BASE_URL` instead of `NEXT_PUBLIC_BASE_URL()` @G3root